### DISCUSSION TITLE
Can't install

### DISCUSSION THREAD

**Discussion by MrPretendo** [earlephilhower/arduino-littlefs-upload#44] [Q&A] [answered]
Hello everyone, what can I do? I can't install the tool, I followed the instructions, but it isn´t working.
Am I doing something wrong?

![Captura de pantalla 2024-07-13 043116](https://github.com/user-attachments/assets/c794a760-d6e8-44ac-9c93-a977a7b1aa5e)
![Captura de pantalla 2024-07-13 043147](https://github.com/user-attachments/assets/99b489da-f621-457e-bcba-2717df0300ca)

**Comment by per1234** [comment, marked as answer]
Hi @MrPretendo

>  I followed the instructions

The fact that you shared a screenshot of the Arduino IDE installation folder, and that the screenshot shows you created a `plugins` folder there makes me doubt that you did. Please read the instructions again very carefully to make certain you followed them correctly:

> `C:\Users\<username>\.arduinoIDE\plugins\` on Windows (you may need to make this directory yourself beforehand)

Note that the extension should be installed under `C:\Users\<username>\.arduinoIDE\plugins\`, NOT under `E:\Archivos de programma (x86)\Arduino\plugins`.
